sources/_sector/dex/trades/arbitrum/_sources.yml

```diff
@@ -28,8 +28,18 @@ sources:
       - name: SwapFactory_evt_PairCreated
       - name: ArbswapStableSwapTwoPool_evt_TokenExchange
       - name: ArbswapStableSwapFactory_evt_NewStableSwapPair
+  - name: trader_joe_v2_2_arbitrum
+    tables:
+      - name: LBPair_evt_Swap
+      - name: LBFactory_evt_LBPairCreated
   - name: trader_joe_v2_1_arbitrum
+    tables:
+      - name: LBPair_evt_Swap
+      - name: LBFactory_evt_LBPairCreated
   - name: trader_joe_arbitrum
+    tables:
+      - name: LBPair_evt_Swap
```

Comment:
Here is a duplicated source, we may remove this line or remove the one defined in `sources/trader_joe/arbitrum/trader_joe_arbitrum_sources.yml`

https://github.com/duneanalytics/spellbook/actions/runs/10576035884/job/29301000660?pr=6618#step:8:20

Reply:
Thanks for the reminder 🙏 Should be fixed now.